### PR TITLE
add clickhouse v0.2.0 to registry

### PR DESCRIPTION
**Pull request by BenoitRanque**

### CODE REVIEW

**Review by sordina — APPROVED** [review]
Looks good. Just double checking that we're removing version `v0.1.0`?

File: registry/clickhouse/metadata.json [hunk 1]
```diff
@@ -1,28 +1,41 @@
...
+        "value": "94bacc789b3c3ea5e92b324e43bf054abaef1c0b7ce87d78ea12a34b33d7d6e5"
+      },
+      "source": {
+        "hash": "4ec3b65c4fdbb37896e105ca89f8bdb1c25b277b"
+      }
     }
+  ],
+  "source_code": {
+    "is_open_source": true,
+    "repository": "https://github.com/hasura/clickhouse_ndc/",
```
Comment (daniel-chambers) [inline registry/clickhouse/metadata.json:32]: This URL is different to what's used in the package. I assume that's a mistake?

Comment (BenoitRanque) [reply registry/clickhouse/metadata.json:32]: that's outdated, correcting

**Comment by BenoitRanque** [comment]
> Looks good. Just double checking that we're removing version `v0.1.0`?

Yes

**Pull request merged**